website/_docs30/tutorial/lambda_mode_and_timezone_realtime_olap.md

```diff
@@ -0,0 +1,174 @@
+
+Secondly, in fact you cannot update a normal streaming cube, but you can update a streaming cube which in lambda mode, all you need to prepare is creating a Hive table which mapping to your kafka event.
+
+Thirdly, yes it is, you can achieved this by add *derived time column* like *MINUTE_START*/*DAY_START* etc in your filter condition.
+
+# How to do
+
+### Configure timezone
+We knew message may come from different timezone, but you want the query result should stick to some specific timezone. 
+If you live in some place in GMT+2, please set *kylin.stream.event.timezone=GMT+2* for all Kylin process. 
+
+
+### Create lambda table
+
+You should create a hive table in *default* namespace, and this table should contains all column in your cube's dimension and measure, please 
+be ware of the *derived time column* like *MINUTE_START*/*DAY_START* which you are interested in should be also included if they are appeared in your cube's dimension column. 
+
+Depend on in which granularity level you want to update segment,  you can choose *HOUR_START* or *DAY_START* as partition column of this hive table.
+
+{% highlight Groff markup %}
+use default;
+CREATE EXTERNAL TABLE IF NOT EXISTS lambda_flat_table
+(
+-- event timestamp and debug purpose column
+EVENT_TIME timestamp
+,ROUND bigint COMMENT "For debug purpose, in which round did this event sent by producer"
+,LOCAL_DAY_HOUR string COMMENT "For debug purpose, maybe check timezone etc"
+,LOCAL_MINUTE string COMMENT "For debug purpose, maybe check timezone etc"
+
+-- dimension column on fact table
+,LO_QUANTITY bigint
+,LO_DISCOUNT bigint
+
+-- dimension column on dimension table
+,C_REGION string
+,C_NATION string
+,C_CITY string
+
+,D_YEAR int
+,D_YEARMONTH string
+,D_WEEKNUMINYEAR int
+,D_YEARMONTHNUM int
+
+,S_REGION string
+,S_NATION string
+,S_CITY string
+
+,P_CATEGORY string
+,P_BRAND string
+,P_MFGR string
+
+
+-- measure column  on fact table
+,V_REVENUE bigint
+,LO_SUPPLYCOST bigint
+,LO_REVENUE bigint
+,PROFIT_RATIO double
+
+-- for kylin used
+,MINUTE_START timestamp
+,HOUR_START timestamp
+,MONTH_START date
+)
+PARTITIONED BY (DAY_START date)
+STORED AS SEQUENCEFILE
+LOCATION 'hdfs:///LacusDir/lambda_flat_table';
+{% endhighlight %}
+
+
+### Create streaming cube in Kylin
+The first step is add information of broker list and topic name; 
+after that, you should paste sample message into left and let Kylin auto-detect the column name and column type.
+You may find some data type is not corrected, please change to what its real data type and make sure it is aligned to Hive table.
+
+For example, you should change the data type of event_time from varchar to timestamp.
+And some column name is not the same as Hive Table, so please correct them too, such as from `customer_info_REGION` to `C_REGION`.
```

Comment:
And some column names are...
so please correct them too, such as `customer_info_REGION` to `C_REGION`.